src/tests/cxxfrontend.c

```diff
@@ -351,14 +351,14 @@ void test_types()
     StorageClass stc = STCnothrow|STCproperty|STCreturn|STCreturninferred|STCtrusted;
     TypeFunction *tfunction = TypeFunction::create(args, Type::tvoid, VARARGnone, LINKd, stc);
 
-    assert(tfunction->isnothrow);
-    assert(!tfunction->isnogc);
-    assert(tfunction->isproperty);
-    assert(!tfunction->isref);
-    assert(tfunction->isreturn);
-    assert(!tfunction->isscope);
-    assert(tfunction->isreturninferred);
-    assert(!tfunction->isscopeinferred);
+    assert(tfunction->isnothrow());
+    assert(!tfunction->isnogc());
+    assert(tfunction->isproperty());
+    assert(!tfunction->isref());
+    assert(tfunction->isreturn());
+    assert(!tfunction->isthisscope());
+    assert(tfunction->isreturninferred());
+    assert(!tfunction->isscopeinferred());
```

Comment:
Could you add tests for the setters too?  One ought to be enough, e.g:
```
tfunction->isref(true);
assert(tfunction->isref());
```

Reply:
done